.github/ISSUE_TEMPLATE/suggest_wallet.md

```diff
@@ -6,98 +6,192 @@ labels: "wallet :purse:,"
 
-Before suggesting a wallet, make sure you've read [our listing policy](https://www.ethereum.org/en/contributing/adding-products/).
+If the wallet does meet our list policy, please complete the following information which we need to accurately list the wallet.
 
-Only continue with the issue if your wallet meets the criteria listed there.
+### Project info
 
-If it does complete the following information which we need to accurately list the wallet.
+**Wallet name**
 
-**Is your wallet security tested? Please explain security measures i.e. security audit, internal security team or some other method.**
+  <!-- Please provide the official name of the wallet -->
+
+**Wallet description**
+
+<!-- Please provide a short 1-2 sentence description of the wallet without marketing claims. Avoid claims like “the best Ethereum wallet”. We will ask for revisions if this comes across as marketing-focused. -->
+
+**Wallet logo**
+
+<!-- Please provide a hi-res SVG or transparent PNG -->
+
+**Background color for brand logo**
+
+<!-- Please provide a hex code for the brand color. This will be added to the background of the wallet card. -->
+
+**URL to the project**
+
+<!-- Please provide a URL (e.g. to the website of the wallet) -->
+
+**URL to the documentation**
+
+<!-- Please provide a URL to the documentation -->
+
+**Is your wallet security tested? Please explain security measures i.e. security audit, internal security team, or some other method.**
 
 <!-- Please explain any security measures you have taken to ensure your wallet is secure -->
 
 **When did your wallet go live to users?**
 
-<!-- Please provide an exact or approximate date when your wallet was usable by the public -->
+<!-- Please provide a date when your wallet was usable by the public. Please provide some user metrics for how many users are using this wallet. -->
 
 **Does your wallet have an active development team?**
 
-<!-- Are developers actively working on the wallet? -->
-
-**Is your wallet open-source?**
-
-<!-- If yes, please provide a direct link to the repository, if you can -->
+<!-- Are developers actively working on the wallet? Provide proof that the wallet is actively being worked on (ex. GitHub repo link). -->
 
 **Is your wallet globally accessible?**
 
 <!-- Please list any KYC requirements or geographic limitations of your wallet, if any exist. -->
 
-**Is your wallet custodial, non-custodial, or a hardware wallet?**
+**Is there internationalization support for your wallet?**
+
+<!-- Please list languages that the wallet actively supports -->
+
+**What is the repository for your codebase?**
+
+<!-- If you’re project is open source, please provide a link to the codebase. -->
+
+**What social links are there for the project?**
+
+<!-- Please provide social links for the wallet (Discord, Twitter, etc.) -->
```

Comment:
Are we going to use this information? We should probably ask for exactly what we want if we are.

Reply:
I don't see this explicitly in the design right now. @nloureiro Was there a plan to include these?

For comparison to other areas of the site, we have the socials in the `staking-products.json` for each product, but we ended up not adding these to the product cards that get rendered. Agree if we decide to not include this in the design then we should probably remove the question.

Reply:
Im ok if these arent in v1 of this release, but I do think at some point showing this information would be ideal.

Reply:
> Im ok if these arent in v1 of this release, but I do think at some point showing this information would be ideal.

Then perhaps we ask explicitly? It'll save a bunch of time in the future.

Discord server:
Twitter:
what else?